lib/rsvp/-internal.js

```diff
@@ -204,6 +204,7 @@ export function invokeCallback(settled, promise, callback, detail) {
     if (value === TRY_CATCH_ERROR) {
       failed = true;
       error = value.error;
```

Comment:
The `value = null` looks like my original but failed trying to release, good catch.